From: kernel test robot <lkp@intel.com>
Subject: mm/shmem: vma_needs_copy can be static

mm/memory.c:1238:1: warning: symbol 'vma_needs_copy' was not declared. Should it be static?

Link: https://lkml.kernel.org/r/Ylb0CGeFJlc4EzLk@7ec4ff11d4ae
Fixes: 729c63ce2bbd ("mm/shmem: handle uffd-wp during fork()")
Signed-off-by: kernel test robot <lkp@intel.com>
Reported-by: kernel test robot <lkp@intel.com>
Acked-by: Peter Xu <peterx@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/memory.c~mm-shmem-handle-uffd-wp-during-fork-fix
+++ a/mm/memory.c
@@ -1234,7 +1234,7 @@ copy_p4d_range(struct vm_area_struct *ds
  * false when we can speed up fork() by allowing lazy page faults later until
  * when the child accesses the memory range.
  */
-bool
+static bool
 vma_needs_copy(struct vm_area_struct *dst_vma, struct vm_area_struct *src_vma)
 {
 	/*
_
